<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RewardEditWidget</class>
 <widget class="QWidget" name="RewardEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="RewardNameLineEdit">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>45</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border:1px solid #c020bb;
background-color: rgb(112, 13, 119);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="placeholderText">
    <string>限4个字</string>
   </property>
  </widget>
  <widget class="QLabel" name="RewardLb">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>125</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DelBtn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>180</y>
     <width>73</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	border-image: url(:/image/RewardEditWidget/bt_delete_hover.png);
}
QPushButton
{
border-image: url(:/image/RewardEditWidget/bt_delete.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>73</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	border-image: url(:/image/RewardEditWidget/bt_save_hover.png);
}
QPushButton
{
border-image: url(:/image/RewardEditWidget/bt_save.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>73</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	border-image: url(:/image/RewardEditWidget/bt_cancle_hover.png);
}
QPushButton
{
border-image: url(:/image/RewardEditWidget/bt_cancle.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
